pyfar/classes/audio.py

```diff
@@ -1197,10 +1238,17 @@ def _get_arithmetic_data(data, n_samples, domain):
         else:
             raise ValueError(
                 f"domain must be 'time' or 'freq' but found {domain}")
-
     else:
         data_out = np.asarray(data)
 
+        if data_out.ndim <= len(cshape):
+            # extend by time/frequency axis, scalars are extended to shape (1,)
+            data_out = data_out[..., None]
+        elif cshape != ():
+            # operation includes arrays and audio objects
+            raise ValueError(
```

Comment:
I did not find a test for this - is there one?

Reply:
yes, in ``test_array_broadcasting_errors()``